job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContextSpec.groovy

```diff
@@ -5175,4 +5176,308 @@ class PublisherContextSpec extends Specification {
         where:
         value << [true, false]
     }
+
+    def 'publish deployment to WebLogic with least args'() {
+        when:
+        context.deployToWeblogic()
+
+        then:
+        context.publisherNodes.size() == 1
+        context.publisherNodes[0].name() == 'org.jenkinsci.plugins.deploy.weblogic.WeblogicDeploymentPlugin'
+        context.publisherNodes[0].mustExitOnFailure[0].value() == false
+        context.publisherNodes[0].forceStopOnFirstFailure[0].value() == false
+        context.publisherNodes[0].isDeployingOnlyWhenUpdates[0].value() == false
+    }
+
+    def 'publish one deployment to WebLogic with all required args'() {
+        when:
+        context.deployToWeblogic {
+
+            task {
+                weblogicEnvironmentTargetedName 'test_environment'
+                deploymentName 'myApp'
+                deploymentTargets 'ms_one'
+                builtResourceRegexToDeploy 'myApp.ear'
+                baseResourcesGeneratedDirectory ''
+                taskName 'test_deploy_task'
+
+                jdkName 'JDK 7'
+
+                stageMode WeblogicDeploymentStageModes.BY_DEFAULT
+            }
+
+        }
+
+        then:
+        context.publisherNodes.size() == 1
+        context.publisherNodes[0].name() == 'org.jenkinsci.plugins.deploy.weblogic.WeblogicDeploymentPlugin'
```

Comment:
use `with(context.publisherNodes[0]) { ... }`, see other test specs for examples